lib/ace/edit_session/bracket_match.js

```diff
@@ -37,6 +37,70 @@ var Range = require("../range").Range;
 
 function BracketMatch() {
 
+
+    this.getBracketsCoor = function(position) {
+        if (position.column == 0) return null;
```

Comment:
@dbarashev I thought about it and realized that in the case of frame nothing will work, because only parsed line on which cursor is located. I'll correct this bug.